modules/ipi-install-configuring-networking.adoc

```diff
@@ -28,87 +32,147 @@ $ export PUB_CONN=<baremetal_nic_name>
-    nmcli con up baremetal
-"
+$ nmstatectl apply <path_to_network_yaml>
 ----
-<1> Replace `<con_name>` with the connection name. Replace `x.x.x.x/yy` with the IP address and CIDR for the network. Replace `a.a.a.a` with the network gateway. Replace `b.b.b.b` with the IP address of the DNS server. 
 
 . Optional: If you are deploying with a provisioning network, export the provisioning network NIC name by running the following command:
 +
 [source,terminal]
 ----
-$ export PROV_CONN=<prov_nic_name>
+$ export PROV_CONN=<provisioning_nic_name>
 ----
 
-. Optional: If you are deploying with a provisioning network, configure the provisioning network by running the following command:
-+
-[source,terminal]
-----
-$ sudo nohup bash -c "
-    nmcli con down \"$PROV_CONN\"
-    nmcli con delete \"$PROV_CONN\"
-    nmcli connection add ifname provisioning type bridge con-name provisioning
-    nmcli con add type bridge-slave ifname \"$PROV_CONN\" master provisioning
-    nmcli connection modify provisioning ipv6.addresses fd00:1101::1/64 ipv6.method manual
-    nmcli con down provisioning
-    nmcli con up provisioning
-"
-----
+. Optional: If you are deploying your cluster in a provisioning network, create or edit an NMState YAML file and specify the details in the file:
 +
 [NOTE]
 ====
-The SSH connection might disconnect after executing these steps.
-
-The IPv6 address can be any address that is not routable through the bare-metal network.
+The IPv6 address can be any address that does not route through the bare-metal network.
 
-Ensure that UEFI is enabled and UEFI PXE settings are set to the IPv6 protocol when using IPv6 addressing.
+Ensure that you enabled Unified Extensible Firmware Interface (UEFI) and set UEFI PXE settings for the IPv6 protocol when using IPv6 addressing.
 ====
-
-. Optional: If you are deploying with a provisioning network, configure the IPv4 address on the provisioning network connection by running the following command:
 +
-[source,terminal]
+[source,yaml]
+----
+# ...
+interfaces:
+  - name: provisioning <1>
+    type: bridge
+    state: up
+    ipv6:
+      enabled: true
+      dhcp: false
+      address:
+        - ip: <ipv6_address>
+          prefix-length: 64
+    bridge:
+      port:
+        - name: <physical_interface_name> <2>
+  - name: <physical_interface_name> <3>
+    type: ethernet
+    state: up
+# ...
+----
+<1> Defines the provisioning bridge interface and its static IPv6 address.
+<2> Replace `<physical_interface_name>` with the physical interface that forms part of the bridge interface.
+<3> Details the physical interface that you set as the bridge port.
+
+. Optional: If you are deploying with a provisioning network, configure the IPv4 address on the provisioning network connection creating the JSON object:
++
+[source,json]
 ----
-$ nmcli connection modify provisioning ipv4.addresses 172.22.0.254/24 ipv4.method manual
+{
+  ipv4: {
```

Comment:
This needs to be in YAML in the previous section. 

We have to configure the entire `provisioning` bridge at onec. 

We need to combine the examples somehow.

Reply:
:heavy_check_mark: